src/modules/children.js

```diff
@@ -0,0 +1,32 @@
+const makeChildrenList = (elementsList) => {
+    let elementsCollection = [];
+    for (let element of elementsList) {
+        elementsCollection.push(...Array.from(element.children));
+    }
+    return elementsCollection;
+}
+
+const getChildren = (element) => {
+    if (element.length === undefined) {
+        return this.children;
+    } else {
+        return makeChildrenList(element);
+    }
+}
+
+const filterChildren = (element, optionalSelector) => {
+    const childrenList = Array.from(getChildren(element));
+    const filteredListOfChildren = childrenList.filter((element) => {
+            return element.matches(optionalSelector);
```

Comment:
you can just write `=> element.matches(optionalSelector)` instead of `return` and `{}`